internal/core/data/device_test.go

```diff
@@ -165,7 +162,7 @@ func buildReadings() []contract.Reading {
 }
 
 func handleDomainEvents(bitEvents []bool, wait *sync.WaitGroup, t *testing.T) {
-	until := time.Now().Add(250 * time.Millisecond) //Kill this loop after quarter second.
+	until := time.Now().Add(500 * time.Millisecond) //Kill this loop after quarter second.
```

Comment:
In the commment, It's a 1/2 second now :)

Reply:
just changed the comment